core/templates/core/reports.html

```diff
@@ -24,6 +24,12 @@
         var entriesApiUrl = "{% url 'entry-list' %}";
         var reportsExportUrl = "{% url 'reports-export' %}";
 		var userId = {{ user.id }};
-        riot.mount('reports');
-    </script>
+        riot.mount('reports', {
+            format: {
```

Comment:
Don't need to include opts of the formats, I already included the formats directly on the reports tag